src/screens/SendToken/SendTokenTransaction.js

```diff
@@ -92,29 +95,30 @@ const getTransactionSuccessTitle = (props) => {
   return t('transactions.title.transactionSuccess');
 };
 
-const CancelText = styled(MediumText)`
-  color: ${themedColors.negative};
-  font-size: ${fontSizes.medium}px;
-`;
+function SendTokenTransaction() {
+  const dispatch = useDispatch();
+  const navigation = useNavigation();
+  const fromAddress = useRootSelector(activeAccountAddressSelector);
 
-const ButtonWrapper = styled.View`
-  width: 100%;
-  margin: 0px ${spacing.layoutSides}px 20px;
-`;
+  const isSuccess: boolean = useNavigationParam('isSuccess');
+  const error: string = useNavigationParam('error', '');
+  const transactionPayload: TransactionPayload = useNavigationParam('transactionPayload');
+  const transactionType: ?string = useNavigationParam('transactionType');
+  const goBackDismiss: ?string = useNavigationParam('goBackDismiss');
+  const noRetry: string = useNavigationParam('noRetry');
+  const batchHash: string = useNavigationParam('batchHash', '');
+  const hash: string = useNavigationParam('hash', '');
```

Comment:
`hash` and `batchHash` might not be present at all, maybe we want to keep this way (`string | void`) and check accordingly? wdyt?

Reply:
I support, should probably use `?string` type and avoid empty string default value.